src/octoprint/plugins/classicwebcam/static/css/classicwebcam.css

```diff
@@ -0,0 +1,148 @@
+#webcam_video_container {
```

Comment:
Am I right in assuming this is autogenerated from the `.less` file?

Reply:
Yes, this is directly moved